plugin/trino-plugin-archetype/src/main/resources/archetype-resources/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
This would be fine for creating a standalone plugin as a separate repo, but if I understand the README correctly this archetype helps to quickly create another trino plugin inside an existing trino fork. Why can't we use the existing `mvnw` trino maven wrapper?

Reply:
This is meant to create standalone modules, but adding the Maven wrapper might be out of scope anyway.